blocks/http/test/CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 add_ut_test(qa_HttpBlock)
 target_link_libraries(qa_HttpBlock PRIVATE gr-http)
 
+# Not using ut as it has issues with static objects
+# https://github.com/boost-ext/ut/issues/611
```

Comment:
As above, this should be an easy fix with adding the `boost::ut::cfg<boost::ut::override>.run();` in the unit-tests `main()` method.